shipyard_blueprints/domo/shipyard_domo/cli/export_card_to_file.py

```diff
@@ -25,214 +24,53 @@ def get_args():
     parser.add_argument(
         "--file-type", dest="file_type", choices={"ppt", "csv", "excel"}, required=True
     )
-    parser.add_argument("--developer-token", dest="developer_token", required=False)
+    parser.add_argument("--developer-token", dest="developer_token", required=True)
```

Comment:
I am going to remove the email and password inputs so that users can only log in through a developer token. This is very low risk since this blueprint is not being used. Additionally, the credential test will require all 4 inputs to pass: Developer Token ,Domo Instance, Client ID, Client Secret